stan/math/prim/fun/acosh.hpp

```diff
@@ -23,10 +23,6 @@ inline double acosh(double x) {
     return x;
   } else {
     check_greater_or_equal("acosh", "x", x, 1.0);
-#ifdef _WIN32
-    if (is_inf(x))
-      return x;
-#endif
```

Comment:
This worked fine on my Windows box, but will let Jenkins give the final verdict. Will just revert back if Jenkins flags this.

Reply:
Is this what made the test fail on Jenkins?

Reply:
Yep, will remove once I update the PR. Was hoping it could go away :)